extensions/sql-database-projects/package.json

```diff
@@ -15,7 +15,7 @@
   "activationEvents": [
     "*"
```

Comment:
Side note: avoid using * if you don't need to. if you have commands, just add those as your activation events. vscode will activate you automatically if it needs you to provide data for the tree.